src/app_service/service/community/service.nim

```diff
@@ -556,7 +558,8 @@ QtObject:
       if prev_community.isOwner and not community.isOwner:
         self.events.emit(SIGNAL_COMMUNITY_LOST_OWNERSHIP, CommunityIdArgs(communityId: community.id))
         let response = tokens_backend.registerLostOwnershipNotification(community.id)
-        self.activityCenterService.parseActivityCenterResponse(response)
+        self.events.emit(SIGNAL_RAW_ACTIVITY_CENTER_NOTIFICATIONS,
+          RawActivityCenterNotificationsArgs(activityCenterNotifications: response.result{"activityCenterNotifications"}))
```

Comment:
Just a wondering, Would it make sense to define `activityCenterNotifications` somewhere as a constant to avoid having to write it everytime? Having a constant should also cover the case when we update it so that we don't miss any portion of code. any thoughts?